app/assets/javascripts/discourse/app/components/post/menu/button-wrapper.gjs

```diff
@@ -0,0 +1,19 @@
+import Component from "@glimmer/component";
+
+// eslint-disable-next-line ember/no-empty-glimmer-component-classes
+export default class PostMenuButtonWrapper extends Component {
+  // we need a class component because we need to pass this.args to the config helpers
+
+  <template>
+    <@buttonConfig.Component
+      class="btn-flat"
+      ...attributes
```

Comment:
I don't see anything actually passing attributes in here? IMO it's better to drop the `...attributes` in this case - then it's clear that we don't expect anything to be passed in.

Reply:
We can drop `...attributes` here as I probably just left it as default because I find annoying when I have to open a PR just to add `...attributes` to a component in order to customize it.

But you're right, it's not needed here.

Reply:
Done